src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/AuthenticationService.cs

```diff
@@ -46,7 +46,7 @@ public class AuthenticationService(IMediator messageBus, IDistributedCache cache
     {
       verificationStatus = new Claim("verified", "true");
     }
-    else if (registrant.Profile.Status == StatusCode.Unverified)
+    else if (registrant.Profile.Status is StatusCode.Unverified or StatusCode.PendingforDocuments or StatusCode.ReadyforIDVerification)
```

Comment:
interesting use of is ... didn't know that was possible in C#. 

I think we can remove StatusCode.ReadyForIDVerification because at this stage, the user has uploaded their ID documents. They shouldn't be able to upload more unless requested to by staff.

Reply:
Yeah I think it's confusing because we're trying to reduce 5 contact statuses to a boolean "verified" 

which is confusing. 

if you go up a few lines, you'll see that a user in ReadyforIDVerification would have a claim of 
(verified, "")

so they wouldn't be able to access this endpoint anyhow.